src/app/Http/Controllers/CrudFeatures/SaveActions.php

```diff
@@ -61,13 +74,14 @@ public function setSaveAction($forceSaveAction = null)
 
     /**
      * Redirect to the correct URL, depending on which save action has been selected.
-     * @param  [type] $itemId [description]
-     * @return [type]         [description]
+     *
+     * @param  string  $itemId
+     * @return \Illuminate\Http\Response
      */
     public function performSaveAction($itemId = null)
     {
         $saveAction = \Request::input('save_action', config('backpack.crud.default_save_action', 'save_and_back'));
-        $itemId = $itemId ? $itemId : \Request::input('id');
+        $itemId = $itemId ?: \Request::input('id');
```

Comment:
Oh how I hate the empty `?:` construct.

Given that PHP 7 is a requirement, though, we could evilly change that to `??` I suppose.

Reply:
PHP 7 became a requirement in 5.5, so sadly no.

Reply:
Yes, but PHP 7 became a backpack requirement recently 👿